docsk8s/administering/ls-k8s-stack-monitoring.asciidoc

```diff
@@ -1,14 +1,124 @@
 
-In order to prepare your environment to be production ready, you'll need to set up secure communication between each of your Elastic resources.
+An important step to making your environment production ready is to configure stack monitoring. Monitoring metrics can be sent to an external resource, such as {ess} or {eck}, so that in the event that any components of your environment become unresponsive, your monitoring data is available.
 
-An important step to making your environment production ready is to configure stack monitoring. Monitoring logs and metrics data can be sent to an external resource, such as {ess} or {eck}, so that in the event that any components of your environment become unresponsive, your monitoring data is available.
+To enable {logstash-ref}/monitoring-with-metricbeat.html[Stack Monitoring] for {ls}, you need {metricbeat} to collect {ls} metrics, {es} to store the metrics and {kib} to view the result.
 
-Follow these steps to configure monitoring:
+==== Stack Monitoring with ECK
 
-. Step 1
-. Step 2
-. Step 3
\ No newline at end of file
+NOTE: The prerequisite of the example is having ECK installed. Checkout <<qs-set-up>>
+
+For these examples, we will be modifying the Beats stack monitoring link:https://github.com/elastic/cloud-on-k8s/blob/main/config/recipes/beats/stack_monitoring.yaml[recipe] from the ECK examples, which initiates a production {es} cluster, a monitoring {es} cluster, {filebeat}, {metricbeat}, a production Kibana and a monitoring Kibana. It monitors {es} and Kibana and send metrics to monitoring cluster.
+We use {metricbeat-ref}/configuration-autodiscover.html[autodiscover] to config monitoring for multiple {ls}.
+
+* <<ls-k8s-monitor-config-metricbeat>>
+* <<ls-k8s-monitor-config-ls>>
+* <<ls-k8s-monitor-kibana>>
+
+[float]
+[[ls-k8s-monitor-config-metricbeat]]
+===== Config Metricbeat
+
+To monitor {ls}, add logstash module to the recipe.
+
+[source,yaml]
+--
+apiVersion: beat.k8s.elastic.co/v1beta1
+kind: Beat
+metadata:
+  name: metricbeat
+spec:
+  type: metricbeat
+  version: 8.4.2
+  elasticsearchRef:
+    name: elasticsearch-monitoring <1>
+  config:
+    metricbeat:
+      autodiscover:
+        providers:
+          - type: kubernetes
+            scope: cluster
+            hints.enabled: true
+            templates:
+              - condition:
+                  contains:
+                    kubernetes.labels.app: ls <2>
+                config:
+                  - module: logstash <3>
+                    metricsets:
+                      - node
+                      - node_stats
+                    period: 10s
+                    hosts: "http://${data.host}:9600"
+                    xpack.enabled: true
+...
+--
+
+<1> {metricbeat} sends metrics to `elasticsearch-monitoring` cluster.
+
+<2> {metricbeat} scans for the pods with label `app: ls` to collect {ls} metrics.
+
+<3> {metricbeat} logstash module calls metric endpoint of each {ls} from port `9600` for every `10` seconds.
```

Comment:
We've experienced some rendering problems when callout descriptions have returns between them.